backend/algorithms/errors.json

```diff
@@ -6374,7 +6342,7 @@
             ")",
             ")"
         ],
-        "broke at": "Index 1, BIOC2101,"
+        "broke at": "Index 3, BIOC2201."
```

Comment:
I mean it wasn't gonna working anyway, but ideally it wouldn't break on a full stop